src/controllers/still.required.controller.ts

```diff
@@ -34,13 +36,18 @@ const postRoute = async (req: Request, res: Response, next: NextFunction): Promi
 
     return renderPageWithError(res, errorText, companyName);
   } else {
-    return res.render(STILL_REQUIRED, {
-      companyName,
-      templateName: STILL_REQUIRED,
-    });
+    await addDecisionToSession(req.body.stillRequired, req.chSession);
+    const companyNumber = getPromiseToFileSessionValue(req.chSession, COMPANY_PROFILE).companyNumber;
+    const url = PROMISE_TO_FILE + "/company/" + companyNumber + APPEND_CONFIRMATION;
```

Comment:
Could use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace
To replace :companyNumber in the route, with the company number we want, would save declaring the APPEND_CONFIRMATION constant.
Something like `const url = CONFIRMATION.replace(":companyNumber", companyNumber")`